src/Scripting/Core/Hosting/CommonMemberFilter.cs

```diff
@@ -13,6 +13,11 @@ internal class CommonMemberFilter : MemberFilter
     {
         public override bool Include(StackFrame frame)
         {
+            if (frame.GetMethod().DeclaringType.FullName.StartsWith("Submission#0"))
```

Comment:
>frame.GetMethod() [](start = 16, length = 17)

If you move this one line below you can use ```method``` variable. Also the DeclaringType might be null, so you need to check for that.